### PR TITLE
chore(i18n,curriculum): processed translations

### PR DESCRIPTION
**Pull request by camperbot**
This PR was opened auto-magically by Crowdin.

### CODE REVIEW

**Comment by ieahleen** [comment]
Node.js CI / Test Upcoming Changes (14.x) (push) failed in Render Conditionally from Props

restarting the tests

**Review by majestic-owl448 — DISMISSED** [review]

**Review by ojeytonwilliams — DISMISSED** [review]

File: curriculum/dictionaries/portuguese/comments.json [hunk 1]
```diff
@@ -1,110 +1,110 @@
 {
-  "hyek8f": "24.44 in Celsius",
```
Comment (ojeytonwilliams) [inline curriculum/dictionaries/portuguese/comments.json:2]: @nhcarrigan any idea how these changes made it into this PR?

Comment (ieahleen) [reply curriculum/dictionaries/portuguese/comments.json:2]: It seems correct, in crowdin the comments are in the curriculum project
image

Comment (ojeytonwilliams) [reply curriculum/dictionaries/portuguese/comments.json:2]: Oh, wow, I should have realised that.  Thanks @ieahleen.

**Review by ojeytonwilliams — APPROVED** [review]

**Review by raisedadead — APPROVED** [review]

**Pull request merged**